libcontainer/integration/exec_test.go

```diff
@@ -1996,3 +1996,69 @@ func TestCGROUPHost(t *testing.T) {
 		t.Fatalf("cgroup link not equal to host link %q %q", actual, l)
 	}
 }
+
+// Test that a container using user namespaces is able to bind mount a folder
+// that has some part of its path with "---" permissions for others
+func TestBindMountAndUser(t *testing.T) {
+	if _, err := os.Stat("/proc/self/ns/user"); os.IsNotExist(err) {
+		t.Skip("userns is unsupported")
+	}
+	if testing.Short() {
+		return
+	}
+
+	rootfs, err := newRootfs()
+	ok(t, err)
+	defer remove(rootfs)
+
+	temphost, err := ioutil.TempDir("", "mnt1host")
+	ok(t, err)
+	defer os.RemoveAll(temphost)
+
+	dirhost := filepath.Join(temphost, "inaccessible", "dir")
+	err = os.MkdirAll(dirhost, 0x755)
```

Comment:
I guess you meant `0o755` or `0755` here.

Reply:
Definitely yes.